src/Stock.js

```diff
@@ -78,7 +82,6 @@ class Stock extends React.Component {
                 <h1>Stock Watcher</h1>
 
                 <StockForm sendStockFormData={this.receiveStockFormData}></StockForm>
-
```

Comment:
I kind of like these extra spaces to separate the distinct components. So we have the stock form (space) the plot (space) etc.

Reply:
Extra spaces have been added.